src/vmm/src/devices/virtio/net/device.rs

```diff
@@ -142,6 +142,8 @@ pub struct Net {
     /// Only if MMDS transport has been associated with it.
     pub mmds_ns: Option<MmdsNetworkStack>,
     pub(crate) metrics: Arc<NetDeviceMetrics>,
+
+    rx_buffer: IoVecBuffer,
```

Comment:
```suggestion
    tx_buffer: IoVecBuffer,
```
?

Reply:
My bad thanks I'll update in next revision